<map id="django.contrib.gis.db.models.proxy.GeometryProxy.__get__" name="django.contrib.gis.db.models.proxy.GeometryProxy.__get__">
<area shape="rect" id="node2" href="$dd/d20/classdjango_1_1db_1_1models_1_1fields_1_1related_1_1_foreign_related_objects_descriptor.html#ab0a434c263d3b2da7995210854850483" title="django.db.models.fields.related.\lForeignRelatedObjectsDescriptor.__set__" alt="" coords="299,5,562,47"/>
<area shape="rect" id="node3" href="$d4/df1/classdjango_1_1db_1_1models_1_1fields_1_1related_1_1_many_related_objects_descriptor.html#acbed615f81acce83ca3ebe361cb77d87" title="django.db.models.fields.related.\lManyRelatedObjectsDescriptor.__set__" alt="" coords="305,71,557,112"/>
<area shape="rect" id="node4" href="$da/d9d/classdjango_1_1db_1_1models_1_1fields_1_1related_1_1_reverse_many_related_objects_descriptor.html#a932010361e1b4392aa50e53740b8db97" title="django.db.models.fields.related.\lReverseManyRelatedObjectsDescriptor.__set__" alt="" coords="281,136,580,177"/>
</map>
